Packs/DeveloperTools/Integrations/XsoarServerPowershell/XsoarServerPowershell.yml

```
@@ -0,0 +1,39 @@
+category: Messaging
+commonfields:
+  id: XsoarServerPowershell
+  version: -1
+configuration: []
+description: This integration allows interacting with XSOAR server.
```

Change the description
```suggestion
description: Powershell Integration for testing that Powershell support is working as expected.
```

---

Change in release note, descriptions.